bokehjs/src/lib/models/glyphs/webgl/multi_marker.ts

```diff
@@ -85,7 +85,7 @@ export class MultiMarkerGL extends BaseMarkerGL {
     this._widths.set_from_prop(this.glyph.size)
     this._angles.set_from_prop(this.glyph.angle)
 
-    this._marker_types = this.glyph.marker
+    this._marker_types = this.glyph.marker as any // TODO
```

Comment:
This PR is tentatively ready. I still need to figure out what to do about webgl support.

Reply:
Unless this is a quick and obvious solution I'd just punt for now with a follow-on issue or discussion.

Reply:
It's resolved now.